src/clocksi_static_tx_coord_sup.erl

```diff
@@ -28,7 +28,7 @@
 
 -export([init/1]).
 
--define(NUM, 100).
+-define(NUM, 1).
```

Comment:
For my machine, having one supervisor gives higher throughput than having 100 supervisors (3600 compared with 2800). Maybe the multiple supervisor setting will give more throughput on machines with more cores, but our targeting benchmark machine will only have two cores, so I think it is more appropriate to use just 1 here.